egs/public_dataset/jsut-song_ver1/train.sh

```diff
@@ -0,0 +1,48 @@
+# /usr/bin/bash
+
+# Copyright 2020 RUC (author: Shuai Guo)
+
+# . ./path.sh
+
+set -e
+
+stage=3
+ 
+
+if [ $stage -le 1 ]; then
```

Comment:
This version is good, but please follow the template in HTS (a new version of presenting the `run.sh`